src/mono/wasm/features.md

```diff
@@ -2,36 +2,125 @@
-[See also](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md)
 
-# EH - Exception handling
+You can disable it by `<WasmEnableSIMD>false</WasmEnableSIMD>`.
+
+See also WebAssembly proposal [SIMD.md](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md)
+
+Some older devices or operating systems don't have the necessary CPU instructions to make this optimization bring the expected perf boost.
+
+## EH - Exception handling
 Is performance optimization enabled by default. It requires recent version of browser.
-You can disable it by `<WasmEnableExceptionHandling>false</WasmEnableExceptionHandling><WasmBuildNative>true</WasmBuildNative>`.
-[See also](https://github.com/WebAssembly/exception-handling/blob/master/proposals/exception-handling/Exceptions.md)
 
-# BigInt
+You can disable it by `<WasmEnableExceptionHandling>false</WasmEnableExceptionHandling>`.
+
+See also WebAssembly proposal [Exceptions.md](https://github.com/WebAssembly/exception-handling/blob/master/proposals/exception-handling/Exceptions.md)
+
+## BigInt
 Is required if the application uses Int64 marshaling in JS interop.
-[See also](https://github.com/WebAssembly/JS-BigInt-integration)
+See also WebAssembly proposal [JS-BigInt](https://github.com/WebAssembly/JS-BigInt-integration)
+
+## fetch browser API
 
-# fetch browser API
 Is required if the application uses [HttpClient](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient)
+
 NodeJS needs to install `node-fetch` and `node-abort-controller` npm packages.
 
-# WebSocket browser API
+## WebSocket browser API
+
 Is required if the application uses [WebSocketClient](https://learn.microsoft.com/en-us/dotnet/api/system.net.websockets.clientwebsocket)
+
 NodeJS needs to install `ws` npm package.
 
+## Content security policy
+
+dotnet runtime for wasm is CSP compliant starting from .Net 8, except legacy JS interop methods.
+
+In order to enable it, please set HTTP headers similar to `Content-Security-Policy: default-src 'self' 'wasm-unsafe-eval'`
+
+See also [CSP on MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)
+
+## ICU
+
+_TODO_
+
+## Timezones
+
+Browsers don't offer API for working with time zone database and so we have to bring the time zone data as part of the application.
+
+If your application doesn't need to work with TZ, you can reduce download size by `<InvariantTimezone>true</InvariantTimezone>`.
+This requires that you have [wasm-tools workload](#wasm-tools-workload) installed.
+
 # Shell environments - NodeJS & V8
 We pass most of the unit tests with NodeJS v 14 but it's not fully supported target platform. We would like to hear about community use-cases.
-We also use v8 engine version 11 or higher to run some of the tests. The engine is lacking many APIs and features.
\ No newline at end of file
+
+We also use v8 engine version 11 or higher to run some of the tests. The engine is lacking many APIs and features.
```

Comment:
In this case it's not "the engine" lacking many APIs and features but instead the d8 shell, which is what jsvu is installing. So something like this might be better:

`We also use the d8 command-line shell, version 11 or higher, to run some of the tests. This shell lacks most browser APIs and features.`